extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/usecase/model/AssetEntryDto.java

```diff
@@ -0,0 +1,24 @@
+package de.sovity.edc.ext.wrapper.api.usecase.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.media.Schema.RequiredMode;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Map;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder(toBuilder = true)
+public class AssetEntryDto {
+    @Schema(requiredMode = RequiredMode.REQUIRED)
+    private String assetRequestId;
```

Comment:
what is a request id?

Reply:
https://github.com/sovity/edc-extensions/issues/471